### DISCUSSION TITLE
Laravel's built-in redirect utility away() dont work in V3 beta

### DISCUSSION THREAD

**Discussion by arfr** [livewire/livewire#5786] [7. Bugs (archived)]
So the laravel's built-in redirect utility `return redirect()->away('https://www.youtube.com/watch?v=dQw4w9WgXcQ')` don't work in a livewire component, but it does worked in V2 :/

![222005](https://github.com/livewire/livewire/assets/6981586/3cba24b6-d64a-42f3-b456-efa176a3137a)

**Comment by PhiloNL** [comment]
Thanks for reporting! 😄 Please remember it's a beta release so not everything might work yet as it did in v2. I can confirm this doesn't work and created a failing test https://github.com/livewire/livewire/pull/5787

**Comment by killjin** [comment]
![image](https://github.com/livewire/livewire/assets/2804854/e5b8cf08-3c58-497d-ae11-b19685ce1484)
---
PHP:
```php
$this->redirect($url);
```
This default use Alpine.navigate?
